### DISCUSSION TITLE
CPM - CPC - CPD

### DISCUSSION THREAD

**Discussion by HazemElknawy9** [giuliopons/adadmin#114] [General]
Is it possible to make it possible to use the three pricing models - CPM - CPC - CPD for the site, and the user can choose any of them when adding the ad, or choose them all?

**Comment by giuliopons** [comment]
No. I'ts not possible.
The pricing model is bound to a position. So if a position works in CPC mode you can't use it in CPM.

You can create different positions and define a pricing model for each, but when the user choose a position the pricing model can't be changed.
